12/umbraco-cms/tutorials/overview.md

```diff
@@ -16,7 +16,7 @@ These tutorials will take you through the process of building different types of
 
 These tutorials will show you how you can extend the Umbraco backoffice to create a fluid and intuitive editing experience for your content editors.
 
-<table data-view="cards"><thead><tr><th></th><th></th><th data-hidden data-card-cover data-type="files"></th></tr></thead><tbody><tr><td><strong>Creating a custom Dashboard</strong></td><td>Go through the steps of setting up and building a custom dashboard for the Umbraco CMS backoffice.</td><td><a href="../.gitbook/assets/dashboards-image.png">dashboards-image.png</a></td></tr><tr><td><strong>Creating a cusom Property Editor</strong></td><td>Learn how to build custom Property Editors to increase the options for adding content.</td><td><a href="../.gitbook/assets/property-image.png">property-image.png</a></td></tr><tr><td><strong>Custom Views for Block Lists</strong></td><td>Build custom views to be displayed when building content with the Block List editor.</td><td><a href="../.gitbook/assets/block-list-image.png">block-list-image.png</a></td></tr></tbody></table>
+<table data-view="cards"><thead><tr><th></th><th></th><th data-hidden data-card-cover data-type="files"></th></tr></thead><tbody><tr><td><strong>Creating a custom Dashboard</strong></td><td>Go through the steps of setting up and building a custom dashboard for the Umbraco CMS backoffice.</td><td><a href="../.gitbook/assets/dashboards-image.png">dashboards-image.png</a></td></tr><tr><td><strong>Creating a custom Property Editor</strong></td><td>Learn how to build custom Property Editors to increase the options for adding content.</td><td><a href="../.gitbook/assets/property-image.png">property-image.png</a></td></tr><tr><td><strong>Custom Views for Block Lists</strong></td><td>Build custom views to be displayed when building content with the Block List editor.</td><td><a href="../.gitbook/assets/block-list-image.png">block-list-image.png</a></td></tr></tbody></table>
```

Comment:
to keep the docs consistent, can you please make this change to v10 and v11 as well?